src/pages/home/Home.tsx

```diff
@@ -108,42 +107,66 @@ const Home: React.FC = () => {
         <hr className="divider" />
 
         <p>This app is brought to you by:</p>
+
         <IonGrid>
           <IonRow>
             <IonCol size="6">
-              <img
-                alt=""
-                src="../../assets/African Lii logo.svg"
-                onError={(e) => {
-                  e.currentTarget.src = "../../assets/shapes.svg"
-                }}
-                onClick={() => handleSupportersLink('https://africanlii.org/')}
-              />
+              <a href="https://africanlii.org/"
+                 target="_blank"
+                 rel="noopener noreferrer"
+              >
+                <img
+                    alt=""
+                    src="/assets/African Lii logo.svg"
+                    onError={(e) => {
```

Comment:
this onError stuff is a hold over and really isn't necessary, let's start doing away with it.

Reply:
removed